R/bfastlite.R

```diff
@@ -33,16 +33,19 @@
 #'
 #' @export
 
-bfastlite <- function(data, formula = response ~ trend + harmon, order = 3,
+bfastlite <- function(data, formula = response ~ trend + harmon,
+                   order = 3, breaks = "LWZ",
```

Comment:
more of a comment: I'm thinking that perhaps we should reconsider the default order. I don't know why the default is 3, but it feels like most often we try to lower it. I may be wrong, but we should document somewhere why 3 is the default choice. On the other hand, I didn't read the full documentation, so maybe I'm wrong about this as well.

Reply:
Hmm, that is true. Order 3 only makes sense in agricultural areas. For the most part the idea of the order parameter is that if you increase it, and there is no fluctuation in the higher orders, they will be close to zero anyway. But in practice that just leads to overfitting. I think 2 would be a better default, indeed.